packages/web/src/scss/components/Modal/_theme.scss

```diff
@@ -25,14 +25,31 @@ $dialog-border-radius: tokens.$radius-200;
 $dialog-background-color: tokens.$background-basic;
 $dialog-shadow: tokens.$shadow-300;
 
-$dialog-scrollable-height: var(--modal-preferred-height-mobile, min-content);
+$_dialog-scrollable-default-height: min-content;
+$_dialog-scrollable-default-max-height: 600px;
+
+$dialog-scrollable-height: var(--modal-height, #{$_dialog-scrollable-default-height});
 $dialog-scrollable-height-tablet: var(
-    --modal-preferred-height-tablet,
-    var(--modal-preferred-height-mobile, min-content)
+    --modal-height-tablet,
+    var(--modal-height, #{$_dialog-scrollable-default-height})
+);
+$dialog-scrollable-height-desktop: var(
```

Comment:
maybe we should generate this. So when the user has their additional breakpoints it still works. But we can do this in the future as well.